workflows/assembly_analysis_pipeline.nf

```diff
@@ -70,26 +70,47 @@ workflow ASSEMBLY_ANALYSIS_PIPELINE {
     * The first step is to:
     * - Gather some statistics about the assembly
     * - Filter by length
-    * - TODO: Remove human and any other host specified - https://www.ebi.ac.uk/panda/jira/browse/EMG-7487
+    * - Human, PhyX and host-contaminant decontamination
```

Comment:
_contaminant decontamination_ 
maybe something like `Decontamination against human, phyX and host(?)`

Reply:
ahh yeah!